data/src/main/java/org/oppia/android/data/backends/gae/model/GaePlatformParameter.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/** Data class for a Platform Parameter. */
```

Comment:
Nit: platform & parameter should be lowercase here.

Reply:
Done